python/doc/examples/reliability_sensitivity/sensitivity_analysis/plot_sensitivity_wingweight.py

```diff
@@ -0,0 +1,462 @@
+
+# %%
+# Then, we evaluate the outputs corresponding to this design of experiments.
+
+# %%
+outputDesignSobol = m.model(inputDesignSobol)
+
+# %%
+# We estimate the Sobol' indices with the `SaltelliSensitivityAlgorithm`.
+
+# %%
+sensitivityAnalysis = ot.SaltelliSensitivityAlgorithm(
+    inputDesignSobol, outputDesignSobol, sizeSobol)
+
+# %%
+# The `getFirstOrderIndices` and `getTotalOrderIndices` methods respectively return estimates of all first order and total Sobol' indices.
+
+# %%
+print('First order indices:', sensitivityAnalysis.getFirstOrderIndices())
+
+# %%
+print('Total order indices:',sensitivityAnalysis.getTotalOrderIndices())
+
+
+# %%
+# The `draw` method produces the following graph. The vertical bars represent the 95% confidence intervals of the estimates.
+
+# %%
+graph = sensitivityAnalysis.draw()
+graph.setTitle('Sobol indices with Saltelli - wing weight')
+view = otv.View(graph)
+
+# %%
+# We see that several Sobol' indices are negative, that is inconsistent with the theory. Therefore, a larger number of samples is required to get consistent indices
+sizeSobol = 10000
+sie = ot.SobolIndicesExperiment(m.distributionX, sizeSobol)
+inputDesignSobol = sie.generate()
+inputNames = m.distributionX.getDescription()
+inputDesignSobol.setDescription(inputNames)
+inputDesignSobol.getSize()
+outputDesignSobol = m.model(inputDesignSobol)
+
+sensitivityAnalysis = ot.SaltelliSensitivityAlgorithm(
+    inputDesignSobol, outputDesignSobol, sizeSobol)
+    
+sensitivityAnalysis.getFirstOrderIndices()
+sensitivityAnalysis.getTotalOrderIndices()
+
+graph = sensitivityAnalysis.draw()
+graph.setTitle('Sobol indices with Saltelli - wing weight')
+view = otv.View(graph)
+
+# %%
+# It improves the accuracy of the estimation but, for very low indices, Saltelli scheme is not accurate since several confidence intervals provide negative lower bounds.
+
+# %%
+# Now, we estimate the Sobol' indices using Polynomial Chaos Expansion.
+# We first create a Functional Chaos Expansion.
+sizePCE = 200
+inputDesignPCE = m.distributionX.getSample(sizePCE)
+outputDesignPCE = m.model(inputDesignPCE)
+
+multivariateBasis = ot.OrthogonalProductPolynomialFactory(
+    [m.distributionX.getMarginal(i) for i in range(m.dim)])
+selectionAlgorithm = ot.LeastSquaresMetaModelSelectionFactory()
+projectionStrategy = ot.LeastSquaresStrategy(inputDesignPCE, outputDesignPCE, selectionAlgorithm)
+totalDegree = 4
+enumfunc = multivariateBasis.getEnumerateFunction()
+P = enumfunc.getStrataCumulatedCardinal(totalDegree)
+adaptiveStrategy = ot.FixedStrategy(multivariateBasis, P)
+algo = ot.FunctionalChaosAlgorithm(
+    inputDesignPCE, outputDesignPCE, m.distributionX, adaptiveStrategy, projectionStrategy)
+
+algo.run()
+result = algo.getResult()
+print(result.getResiduals())
+print(result.getRelativeErrors())
+
+# %%
+# The relative errors are very low that indicate the good accuracy of the PCE model.

```

Comment:
```suggestion
# The relative errors are very low: this indicates the PCE model has good accuracy.

```

Reply:
Thank you Joseph for your comments. The corresponding example has been modified accordingly.